shenyu-admin/src/main/resources/mappers/rule-condition-sqlmap.xml

```diff
@@ -159,24 +176,24 @@
                 param_value = #{paramValue,jdbcType=VARCHAR},
             </if>
         </set>
-        where id = #{id,jdbcType=VARCHAR}
+        WHERE id = #{id,jdbcType=VARCHAR}
     </update>
 
     <delete id="delete" parameterType="java.lang.String">
-        delete from rule_condition
-        where id = #{id,jdbcType=VARCHAR}
+        DELETE FROM rule_condition
+        WHERE id = #{id,jdbcType=VARCHAR}
     </delete>
 
     <delete id="deleteByRuleIds" parameterType="java.util.List">
-        delete from rule_condition
-        where rule_id in
+        DELETE FROM rule_condition
+        WHERE rule_id IN
```

Comment:
`WHERE` should be right-aligned with `FROM`

Reply:
copy

> pls modify name of pr by https://shenyu.apache.org/community/issue-pr . input about #3088 in your comment

copy that